java/src/main/java/api/example/java/Config.java

```diff
@@ -0,0 +1,89 @@
+package api.example.java;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.Base64Utils;
+import org.springframework.util.StringUtils;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@Configuration
+public class Config {
+
+	@Value("${climate.login.server}")
+	public String loginServer;
+
+	@Value("${climate.login.path}")
+	public String loginPath;
+
+	@Value("${climate.token.server}")
+	public String tokenServer;
+
+	@Value("${climate.token.path}")
+	public String tokenPath;
+
+	@Value("${climate.api.server}")
+	public String apiServer;
+
+	@Value("${CLIMATE_APP_ID}")
+	public String appKey;
```

Comment:
I would name this clientId (preferred) or appId.

Reply:
yes